src/common/components/img-gallery/index.jsx

```diff
@@ -1,36 +1,66 @@
-import React from 'react'
+import React, { useEffect } from 'react'
+import { connect } from 'react-redux'
 
-import style from 'common/components/scss/ImgGallery.module.scss'
+import { requestImagesGallery } from 'redux/actions'
+import { array, bool, func, string, number } from 'prop-types'
+
+import { Preloader, Search, Pagination, SliderYears } from 'ui-kit'
 
 import noImage from 'assets/images/no-image.png'
+import style from 'common/components/scss/ImgGallery.module.scss'
 
```

Comment:
but we discussed a lot of times to group imports together :)

please group them.... is there a problem with **eslint** or **prettier** ? we spent a lot of time to install **eslint** and **prettier** in our project - and now these import looks like shit.

If there's a problem with **eslint** - we need to fix that, did you run eslint before commit and push ? it should give you some errors saying that imports are not correctly grouped